scripts/reduction_workflow/instruments/sans/sns_command_interface.py

```diff
@@ -5,6 +5,7 @@
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
 # pylint: disable=invalid-name,unused-import
+# ruff: noqa: F403   # Allow wild imports
```

Comment:
Why is this one ignored?

Reply:
I ignored this file because `sns_command_interface` is used a bit like an __init__ file in that it imports everything, and then other files attempt to import things from `sns_command_interface` - so I was not confident I would find all the imports across the codebase to put into `sns_command_interface`